code/datums/mutantraces.dm

```diff
@@ -494,6 +518,15 @@
 	say_filter(var/message)
 		return replacetext(message, "s", stutter("ss"))
 
+	disposing()
+		if(ishuman(mob))
+			var/datum/component/C = mob.GetComponent(/datum/component/consume/organpoints)
+			if (C) C.RemoveComponent(/datum/component/consume/organpoints)
+			var/mob/living/carbon/human/L = mob
+			L.abilityHolder.removeAbility(/datum/targetable/lizardAbility/colorshift)
+			L.abilityHolder.removeAbility(/datum/targetable/lizardAbility/colorchange)
```

Comment:
I wonder if just removing the whole lizard ability holder would work instead. Haven't checked the code but it would likely be cleaner than removing the abilities themselves and leaving the holder dangling there.

Reply:
I tried doing that, but I couldn't figure put how to make it give the ability holder back if someone stops being a lizard then goes back to being one. Or if they are a lizard and they have their mutantrace set to lizard for whatever reason.